### ISSUE TITLE
frozen at step 14 maxbin

### ISSUE THREAD

**Issue by yjiakang** [jtamames/SqueezeMeta#259]
Hello, step 14 has been running long time (~1 month), can you give some suggestions? Here is the syslog file.
[syslog.zip](https://github.com/jtamames/SqueezeMeta/files/6130149/syslog.zip)
Thanks in advance. 
Best,
Jk Yin

**Comment by Shellfishgene** [comment]
See here: https://github.com/jtamames/SqueezeMeta/issues/69

**Comment by yjiakang** [comment]
@Shellfishgene Thanks, may be it used only one core though I set threads to 36. So can you instruct me to restart it with the parameter --no-maxbin? Edit the SqueezeMeta_conf.pl?
Best

**Comment by Shellfishgene** [comment]
Yes, I would try to change `$nomaxbin` to `1` in the config file and restart from step 16.

In my case I now used [VAMB](https://github.com/RasmussenLab/vamb) instead of maxbin2. I ran VAMB outside of SqueezeMeta. The bin fasta files from VAMB can just be copied into the `project/results/maxbin/` directory, renamed from `.fna` to `.fa`, and then SqueezeMeta can be resumed from step 16. This seems to work, but I did not check the results of DASTool properly yet.
